### ISSUE TITLE
brace-style fails to catch this pattern

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#758]
Brace-style should fail in this case regardless of the option used:

``` js
function foo() { return; }
```

Both of the brace styles supported require curly braces on separate lines.
